### ISSUE TITLE
Problems with drush?

### ISSUE THREAD

**Issue by cosmicdreams** [gabesullice/drupal-contributor-project#1]
Hey I just tried to require drush/drush and ran into a TON of conflicts.  I could post you a wall of text but just wanted to check in if you were aware of any conflicts with requiring drush (via composer) with this.

Discovered while prepping a module for D9 that includes a custom drush command and using drupal-check.

**Comment by gabesullice** [comment]
Hey! Thanks for checking out the project!

I was not aware of that, but I was also not expecting anyone to try this out yet 😝

I have been doing some debugging with colleagues and myself. However, that definitely _should_ work, so I'll be looking into it soon. Thanks for the bug report! (also, PRs welcome ;))

**Comment by gabesullice** [comment]
Confirmed this myself by running `composer require drush/drush` and got a wall of conflicts.

**Comment by gabesullice** [comment]
After removing my lock file, I was able to install Drush 10. @cosmicdreams can you try that?

**Comment by cosmicdreams** [comment]
No, after removing my .lock file and tried to compose require drush/drush I got the same wall of conflicts.

**Comment by cosmicdreams** [comment]
I was able to find a way to make it work though.  With this:

`composer require drush/drush --no-update && composer update`

**Comment by cosmicdreams** [comment]
That's what I typically have to do for lightning updates so it's not surprising.
